docs/atomic-pull.1.md

```diff
@@ -11,7 +11,7 @@ atomic-pull - fetch an image locally
 IMAGE
 
 # DESCRIPTION
-**atomic pull**, will fetch a remote image and store it locally.
+**atomic pull**, will fetch a remote image and store it locally. Can pull images from ostree, oci, docker, and dockertar. Atomic pulls from docker by default.
 
```

Comment:
This should be changed accordingly to say that by default it pulls from a Docker registry